ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+| __Edge contacts:__   | There are two main ways to resolve edge contacts, the first is to create a wider edge so that there is a common surface, the second is to cut out the edge so that there is are no shared features between the two components. This is dependent on the nature of the model and the analysis.  | - Go to the design tab, sketch section, and select the line option. Create a rectangle that is perpendicular to the edge contact and shares an edge with the top edge of the face that contains the edge contact. <br/></br> - Select the rectangle using the "Select" tool and then click "Pull". </br></br> 1. To create a gap select "Cut" from the "Options-Pull" toolbar and then either drag the cut to the bottom of the component, use the "Up To" option and select the bottom face, or use the "Ruler" option and make the pull 100cm. <br/></br> 2. To create a wider edge select "Add" and -similarly to cutting- either drag the cut to the bottom of the component, use the "Up To" option and select the bottom face, or use the "Ruler" option and make the pull 100cm. This method will create interference between the new volume and old which can be resolved using the "Interference" tool.  | ![RectangleEdgeContact](https://github.com/NybergWISC/model_development_workflow/blob/master/PicturesSnapshots/ErrorBlocks/RectangleEdgeContact2.JPG "RectangleEdgeContact") |
+| __Unnecessary details:__   |  In this particular case the physical threads are an unnecessary detail as they will cause a significant amount of computational load, but have very little effect on the final results. Which details are important to keep and which to delete is dependent on the nature of the model and the analysis.    | - This step can be completed in any CAD program and is more of a general suggestion to be considered continuously during the design phase. <br/></br> - In SpaceClaim one can use a circular pull to flatten the threads off in a similar way to fixing an edge contact. | ![ScrewedUpDetails](https://github.com/NybergWISC/model_development_workflow/blob/master/PicturesSnapshots/ErrorBlocks/DetailCloseUp.JPG "ScrewedUpDetails") |
+
+<details><p><summary>Notes</summary>
+- Most often in real models unnecessary details must be determined intuitively and visually while overlaps, edge contacts, and small gaps cannot be found visually.
+<br/></br>
+- For all of these edits one must work closely with the creator of the model/experiment to determine which complications are important to the end result and which are able to be changed.
+</p></details>
+
+### In Cubit/Trelis
+_Cubit and Trelis are basically the same CAD software with Trelis being the commercial version. They are most useful for the meshing of geometries and assigning materials to volumes._
+
+| Steps  | Graphic User Interface  | In Command Line  |

```

Comment:
I would make a note above this table that the procedure is described for both the GUI and command line, but a user only needs to do each step in one or the other, not both.